flink-connectors/flink-connector-cassandra/pom.xml

```diff
@@ -176,5 +176,23 @@ under the License.
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-table_2.10</artifactId>
+			<version>${project.version}</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-scala_2.10</artifactId>
```

Comment:
Please point me to the test that actually uses the scala dependendies, because i can't find any.